eahub/templates/base_bs5.html

```diff
@@ -21,3 +21,18 @@
     <link  rel="stylesheet" href="{% static 'dist/global_bs5.css' %}">
   {% endif %}
 {% endblock static %}
+
+{% block messages %}
+
+{% if messages %}
+<div class="container mt-2 w-75">
+  {% for message in messages %}
+  <div class="alert {% for tag in message.tags %} alert-{{ tag }} {% endfor %} alert-dismissible fade show" role="alert">
+    {{ message }}
```

Comment:
There is a problem with including messages into the base template on the ```accounts/confirm-email/``` page appearing after signing up since we have hardcoded the message in that case ("Confirmation email sent to <email-address>") into the html page for ```accounts/confirm-email/```.  

Please remove that hard coded message so that only the dynamic message shows.

Another problem is that for this message the bootstrap class "alert-info" is shown, but for some reason no styling is applied here. Please do either of:
- Create ```hub-info``` class with a neutral background-color  
- Figure out why ```alert-info``` is not applied and fix it

![image](https://user-images.githubusercontent.com/15908208/129342444-a13b54bf-69d8-4b4a-b715-901d1d26551d.png)